README.rst

```diff
@@ -99,6 +99,14 @@ Config Settings
     # (optional, default: constructed from ckanapi version and url)
     ckan.syndicate.user_agent = My User Agent
 
+    # Try to preserve dataset's organization
+    # (optional, default: false)
+    ckan.syndicate.replicate_organization = boolean
+
+    # Try the username of user, whose api_key is used
```

Comment:
I had meant for this comment to read:
`# The username whose api_key is used`

However, (and sorry to move the goalposts here) I think this needs a bit more explanation:

```
# If the dataset already exists on the target CKAN instance, the dataset will be updated
# only if this option is set and its creator matches this user name
# (optional, default: None)
```